x-pack/plugins/apm/ftr_e2e/cypress/support/types.d.ts

```diff
@@ -8,7 +8,7 @@
 declare namespace Cypress {
   interface Chainable {
     loginAsReadOnlyUser(): void;
-    loginAsSuperUser(): void;
+    loginAsPowerUser(): void;
```

Comment:
Thanks for removing the superuser. We really should only use the two others. 
Going forward I think we can replace our custom roles with the built in roles `viewer` and `editor`.